packer/scripts/ubuntu/update.sh

```diff
@@ -16,12 +16,16 @@ sed -i.bak 's/^Prompt=.*$/Prompt=never/' /etc/update-manager/release-upgrades;
 apt-get -y update;
 
 # update package index on boot
-cat <<EOF >/etc/init/refresh-apt.conf;
-description "update package index"
-start on networking
-task
-exec /usr/bin/apt-get update
-EOF
+# Upstart not incl in 20.04 - init dir no longer present in 20.04
+# Does this need to continue to exist? If so, this file needs to be created in
+# a different location.
+
+#cat <<EOF >/etc/init/refresh-apt.conf;
+#description "update package index"
+#start on networking
+#task
+#exec /usr/bin/apt-get update
```

Comment:
If I recall correctly,  this is intended to somehow have an updated system when you boot the image. We could merge this as is, and create an issue in order to address that later...